render.yaml

```
@@ -0,0 +1,11 @@
+services:
+  - type: web
+    name: i7-game-guide
+    runtime: node
+    buildCommand: npm install && npm run build
+    startCommand: npm start
+    envVars:
+      - key: NODE_ENV
+        value: production
+      - key: DATABASE_URL
+        sync: false # This will be set from Render dashboard or via API
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)